apps/api/infra/api.ts

```diff
@@ -177,10 +177,9 @@ export const serviceSetup = (services: {
       PKPASS_API_KEY: '/k8s/api/PKPASS_API_KEY',
       PKPASS_API_URL: '/k8s/api/PKPASS_API_URL',
       PKPASS_SECRET_KEY: '/k8s/api/PKPASS_SECRET_KEY',
-      XROAD_BASE_URL: '/k8s/application-system-api/XROAD_BASE_PATH',
-      XROAD_PROVIDER_ID:
+      PAYMENT_XROAD_PROVIDER_ID:
         '/k8s/application-system-api/PAYMENT_XROAD_PROVIDER_ID',
-      XROAD_CLIENT_ID: '/k8s/application-system-api/XROAD_CLIENT_ID',
+      PAYMENT_XROAD_CLIENT_ID: '/k8s/application-system-api/XROAD_CLIENT_ID',
```

Comment:
we probably just want to remove this.. we are currently not using it...

Reply:
outdated